docs/developer/dependencies.md

```diff
@@ -12,15 +12,22 @@ The python requirements are defined in `requirements.in` and `requirements_dev.i
 (Re)Generate the requirements files:
 
 ```
-$ docker run --rm -v $(pwd):/code standards-lab_standards-lab pip-compile
+$ docker run --rm -v $(pwd):/code rhiaro/pip-tools <selected requirements.in file>
```

Comment:
I guess this path was just for your own?

Reply:
No, this is the image tag on docker hub. I'm going to move it to ODS at some point and will update the docs when I do.